app/views/pages/ResultDetailView.scala.html

```diff
@@ -39,7 +39,7 @@
 
 <br/>
 
-@h2(Messages("quick_calc.result.total_deductions.info"))
+@h2(Messages("quick_calc.result.total_deductions.info_a") + " " + label.toLowerCase + "ly " + Messages("quick_calc.result.total_deductions.info_b"))
```

Comment:
Pretty sure there are labels with ly. ALSO you need this in a message file to we can support welsh